index.js

```diff
@@ -88,7 +91,7 @@ module.exports = function(type, options) {
       contents = _.template(opts.wrapper)(context);
     }
 
-    if (type === 'amd') { contents = makeAMD(contents, opts); }
+    if (type === 'amd') { contents = makeAMD(contents, opts, file.path); }
```

Comment:
Can you update all of the `make<Type>` functions to receive this new argument as the second argument?